.github/workflows/python-min-deps.yml

```diff
@@ -74,12 +49,15 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
           persist-credentials: false
+          fetch-depth: 2
           submodules: "recursive"
 
-      - name: Set up Python ${{ env.PYTHON_VERSION }}
+      - name: Set Python version vars
+        uses: ./.github/actions/build_info
+      - name: Set up Python ${{ env.PYTHON_MAX_VERSION }}
```

Comment:
Why are we setting up max version? This workflow is supposed to run the oldest version we support, to cover the oldest dependencies possible.

Reply:
Copy-paste mistake. Fixed.